include/nanospline/BezierPatch.h

```diff
@@ -550,8 +550,7 @@ class BezierPatch final : public PatchBase<_Scalar, _dim>
             ThisType patch = subpatch(uv_min(0), uv_max(0), uv_min(1), uv_max(1));
 
             // 4. repeat recursively
-            UVPoint uv = patch.approximate_inverse_evaluate(
-                p, num_samples, uv_min(0), uv_max(0), uv_min(1), uv_max(1), level - 1);
+            UVPoint uv = patch.approximate_inverse_evaluate(p, num_samples, 0, 1, 0, 1, level - 1);
```

Comment:
Good catch! Splines don't need this fix because their domain changes as we split the patches, right?

Reply:
That is right.  Only Bezier and RationalBezier require this.